bitnami/flux/values.yaml

```diff
@@ -1112,6 +1118,9 @@ sourceController:
   ## @param sourceController.args Override default container args (useful when using custom images)
   ##
   args: []
+  ## @param sourceController.automountServiceAccountToken Mount Service Account token in pod
+  ##
+  automountServiceAccountToken: true
```

Comment:
shouldn't it be set to `false` instead?

Reply:
They require access to the K8s API so they need to mount the SA token